llvmlite/binding/newpassmangers.py

```diff
@@ -0,0 +1,305 @@
+from ctypes import (c_bool, c_int)
+from llvmlite.binding import ffi
+from llvmlite.binding.initfini import llvm_version_info
+
+llvm_version_major = llvm_version_info[0]
+
+
+def create_new_module_pass_manager():
+    return NewModulePassManager()
+
+
+def create_new_function_pass_manager():
+    return NewFunctionPassManger()
+
+
+def create_pass_builder(tm, pto):
+    return PassBuilder(tm, pto)
+
+
+def create_pipeline_options():
+    return PipelineTuningOptions()
+
+
+class NewModulePassManager(ffi.ObjectRef):
+
+    def __init__(self, ptr=None):
+        if ptr is None:
+            ptr = ffi.lib.LLVMPY_CreateNewModulePassManager()
+        ffi.ObjectRef.__init__(self, ptr)
```

Comment:
I think it would be more idiomatic to do:

```suggestion
        super().__init__(ptr)
```

(the old pass manager is this way because it was written when Python 2 was still supported, where you couldn't do this).